frontend/src/components/activePage/TableWorkspaceLogic/TableWorkspaceLogic.tsx

```diff
@@ -135,6 +149,7 @@ const TableWorkspaceLogic: FC<ITableWorkspaceLogicProps> = ({ ...props }) => {
         },
       });
     }
+    return () => unsubscribe?.();
```

Comment:
Would move this one inside the if, so that it's not necessary to declare an empty one at the beginning.